frontend/components/toolPage/toolPageConda.html

```diff
@@ -0,0 +1,4 @@
+<div style="background:#eee; padding:1pc; border-radius:5px">
+    to install with <a href="https://conda.io/docs/user-guide/tasks/manage-pkgs.html">Conda:</a>
+        <div style="margin-left:2.5pc; margin-top:1pc"><code style="color:#666">conda install -c {{ software.conda_channel }} {{ software.conda_package }}</code></div>
```

Comment:
I like that! But what about channel dependencies? Do install a bioconda pkgs you also need to have conda-forge activated. Should conda_channel be a ordered list instead?

Reply:
Good call! Could just be a comma-separated list of channel names, and the installation instructions could then show how to add those channels.